tests/test_fips.py

```diff
@@ -350,3 +367,25 @@ def test_nss_firefox_cert(container_per_test: ContainerData) -> None:
     c.check_output(
         'certutil -R -k rsa -g 2048 -s "CN=Daniel Duesentrieb3,O=Example Corp,L=Mountain View,ST=California,C=DE" -d "${PWD}/nssdb" -o cert9.cer -f password.txt -z seedfile.dat',
     )
+
+
+@pytest.mark.parametrize(
+    "container", FIPS_OPENJDK_IMAGES, indirect=True,
+)
+def test_openjdk_cert(container: ContainerData) -> None:
+    c = container.connection
+    # Install required packages for the test
+    c.check_output(
+        f"zypper -n install git-core java-$JAVA_VERSION-openjdk-devel"
+    )
+
+    # Clone, compile and run the scripts to check the FIPS provider
+    c.check_output(
+        "cd ~; git clone -q https://github.com/ecki/JavaCryptoTest; cd JavaCryptoTest/src/main/java/"
+    )
+    c.check_output(
+        'javac net/eckenfels/test/jce/JCEProviderInfo.java; java -cp ~/JavaCryptoTest/src/main/java/ net.eckenfels.test.jce.JCEProviderInfo | grep "Listing all JCA Security Providers"'
+    )
+    c.check_output(
+        'cp /src/Tcheck.java .; javac Tcheck.java; java Tcheck | grep "1. SunPKCS11-NSS-FIPS"'
```

Comment:
You're by default in `WORKDIR`, which is `/src/` so the copy is a nop:
```suggestion
        'javac Tcheck.java; java Tcheck | grep "1. SunPKCS11-NSS-FIPS"'
```

Reply:
Changed